.gitignore

```diff
@@ -7,4 +7,6 @@ coverage
 .nyc_output
 *.snap
 .DS_Store
-.idea
\ No newline at end of file
+.idea
+*.swp
+*.swo
```

Comment:
Hey @dessHub, Were these changes related to this ticket?

Reply:
Yes, my vim editor generates some temporary files so I had to ignore them because they always end up on the repo.